src/popup/router/pages/Names/List.vue

```diff
@@ -22,7 +22,8 @@
         <span class="address">{{ owner }}</span>
         <CheckBox
           :value="autoExtend"
-          @input="value => $store.commit('names/setAutoExtend', { index, value })"
+          prevent
+          @click.native="$store.commit('names/setAutoExtend', { index, value: !autoExtend })"
```

Comment:
```suggestion
          @input="value => $store.commit('names/setAutoExtend', { index, value })"
          @click.native.prevent
```

I think CheckBox's `prevent` shouldn't be used, can't it work this way? To use input event with CheckBoxes looks more semantic.

Reply:
And let's don't put clickable inside clickable the next time...

Reply:
I have this issue in firefox both in extension and web-wallet and I've tried this solution but it doesn't work 🤷